libs/npm/core/src/components/show/Show.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export interface IShowProps {
+  /** Whether the children will be visible. */
+  visible: boolean;
+  /** Children nodes to display. */
+  children: React.ReactNode;
+}
+
+/**
+ * A Show component simply wraps conditional elements in a component.
+ * Reducing conditional logical statements within JSX.
+ * @param param0 Component properties.
+ * @returns A new instance of a Show component.
+ */
+export const Show: React.FC<IShowProps> = ({ visible, children }) => {
```

Comment:
@jdtoombs I thought you had already created something like this, but I couldn't find it.  So I added it.

Reply:
That was just for the modal I believe!